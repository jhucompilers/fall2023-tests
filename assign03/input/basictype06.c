int main(void) {
  // "short" and "short int" are the same
  short int si;
  // "long" and "long int" are the same
  long int li;

  si = -10;
  li = -11;

  return 0;
}
