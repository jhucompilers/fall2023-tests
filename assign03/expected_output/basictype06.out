0|main|function|function () returning int
2|si|variable|short
2|li|variable|long
